ADOPTERS.md

```diff
@@ -2,4 +2,5 @@
 | ------------ | ------- | ------------------ |
 | [Spotify](https://www.spotify.com) |[@alund](https://github.com/alund)| Main interface towards all of Spotify's infrastructure and technical documentation.|
 | [bol.com](https://www.bol.com) |[@RoyJacobs](https://github.com/RoyJacobs)| Initial work being done to unify platform tooling.|
-| [DFDS](https://www.dfds.com) |[@carlsendk](https://github.com/carlsendk)| V2 self-service platform.|
\ No newline at end of file
+| [DFDS](https://www.dfds.com) |[@carlsendk](https://github.com/carlsendk)| V2 self-service platform.|
+| [DFDS](https://www.roku.com) |[@timurista](https://github.com/timurista)| Initial work on Cloud engineering service platform.|
```

Comment:
Copy-paste error here @timurista -- should be Roku

Reply:
lol yeah thanks